### ISSUE TITLE
Request: Web clipper

### ISSUE THREAD

**Issue by hgaronfolo** [Laverna/laverna#270]
I really like the look and feel of Laverna! Great job! I am pretty sure that I would make the change from Evernote to Laverna if Laverna had a Web Clipper like the one on Evernote. I use this functionality everyday and can't live without it.

**Comment by zuphilip** [comment]
There seems to be already a webclipper for Firefox: https://github.com/Laverna/webclipper . However, at least for me it is not working currently. AFAIK mozilla changed quite a lot stuff, which also influences extensions. Any plans to fix the webclipper?

**Comment by wwwredfish** [comment]
We are planning to fix/improve webclipper after we implement synchronization and collaboration.

**Comment by hgaronfolo** [comment]
Any news on a web clipper for Chrome? Is there a place where I can see the general road map for Laverna?

**Comment by yarden** [comment]
I also would completely switch over if there was a web clipper function in Laverna (for either Firefox or Chrome), which I also use everyday.

**Comment by jteib** [comment]
I agree!!  Currently waiting to switch over myself, once there is web clipper (preferably Chrome) functionality.
